packages/graphql/src/translate/cypher-builder/variables/Literal.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as CypherBuilder from "../CypherBuilder";
+import { TestClause } from "../utils/TestClause";
+
+describe("Literal", () => {
+    it("Serialize string value", () => {
+        const stringLiteral = new CypherBuilder.Literal("hello");
+
+        const testClause = new TestClause(stringLiteral);
+
+        const queryResult = testClause.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`"\\"hello\\""`);
+    });
+
+    it("Serialize boolean value", () => {
+        const booleanLiteral = new CypherBuilder.Literal(true);
+
+        const testClause = new TestClause(booleanLiteral);
+
+        const queryResult = testClause.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`"true"`);
+    });
+
+    it("Serialize number value", () => {
+        const numberLiteral = new CypherBuilder.Literal(5);
+
+        const testClause = new TestClause(numberLiteral);
+
+        const queryResult = testClause.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`"5"`);
+    });
+
+    it("Serialize array", () => {
+        const literal = new CypherBuilder.Literal(["hello", 5, "hello"]);
+
+        const testClause = new TestClause(literal);
+
+        const queryResult = testClause.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`"[\\"hello\\", 5, \\"hello\\"]"`);
```

Comment:
question (non-blocking): Doesn't the `5` also need to be in quotes `"5"`? (It's the case on line 48)

Reply:
Numbers and booleans are not quoted, as those are directly translated in cypher.

E.g.
```
MATCH(m:Movie)
WHERE m.title = "Matrix"
AND m.released = 1999
```

Reply:
I think that it's not the case in the line 48, the cypher generated is a string but `5` is not, if it was a string it would have been wrapped as `"\\"5\\""` as for the test line 30.